### PR TITLE
fix for `-cvsStats` option of snpEff

### PR DESCRIPTION
**Pull request by nekrut**
Additional fixes for snpEff:

 - Added `-cvsStats` option
- Modified multiQC to render SnpEff reports

### CODE REVIEW

File: tool_collections/snpeff/snpEff.xml [hunk 1]
```diff
@@ -352,9 +359,18 @@
...
-            <filter>generate_stats == True</filter>
+
+
+
+
+        <data name="statsFile" label="${tool.name} on ${on_string} - stats">
+            <filter>reports != "nothing"</filter>
+            <change_format>
+                 <when input="reports" value="html" format="html" />
+                  <when input="reports" value="csv" format="csv" />
```
Comment (bgruening) [inline tool_collections/snpeff/snpEff.xml]: indentation

File: tools/multiqc/multiqc.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <tool id="multiqc" name="MultiQC" version="@WRAPPER_VERSION@.1">
     <description>aggregate results from bioinformatics analyses into a single report</description>
     <macros>
-        <token name="@WRAPPER_VERSION@">1.3</token>
```
Comment (bgruening) [inline tools/multiqc/multiqc.xml]: @nekrut can this be in a separate PR. @bebatut will work soon on updating and fixing multiQC.

**Comment by nekrut** [comment]
This has been resolved in #1803 and @bebatut took care of multiqc, so closing this one.
